packages/grafana-ui/src/components/Card/Card.tsx

```diff
@@ -0,0 +1,245 @@
+import React, { cloneElement, FC, HTMLAttributes, ReactElement, ReactNode, useCallback, useMemo } from 'react';
+import { css, cx } from 'emotion';
+import { GrafanaTheme } from '@grafana/data';
+import { useTheme, styleMixins, stylesFactory } from '../../themes';
+import { Tooltip, PopoverContent } from '../Tooltip/Tooltip';
+import { OnTagClick } from '../Tags/Tag';
+import { TagList } from '../Tags/TagList';
+
+/**
+ * @public
+ */
+export interface ContainerProps extends HTMLAttributes<HTMLOrSVGElement> {
+  /** Content for the card's tooltip */
+  tooltip?: PopoverContent;
+}
+
+const CardContainer: FC<ContainerProps> = ({ children, tooltip, ...props }) => {
+  return tooltip ? (
+    <Tooltip placement="top" content={tooltip} theme="info">
+      <div {...props}>{children}</div>
+    </Tooltip>
+  ) : (
+    <div {...props}>{children}</div>
+  );
+};
+
+/**
+ * @public
+ */
+export interface CardInnerProps {
+  href?: string;
+}
+
+const CardInner: FC<CardInnerProps> = ({ children, href }) => {
+  const theme = useTheme();
+  const styles = getCardStyles(theme);
+  return href ? (
+    <a className={styles.innerLink} href={href}>
+      {children}
+    </a>
+  ) : (
+    <>{children}</>
+  );
+};
+
+/**
+ * @public
+ */
+export interface Props extends ContainerProps {
+  /** Main heading for the Card **/
+  heading: ReactNode;
+  /** Additional data about the card. If array is supplied, elements will be rendered with vertical line separator */
+  metadata?: ReactNode | ReactNode[];
+  /** Card description text */
+  description?: string;
+  /** List of tags to display in the card */
+  tags?: string[];
+  /** Optional callback for tag onclick event */
+  onTagClick?: OnTagClick;
+  /** Indicates if the card and all its actions can be interacted with */
+  disabled?: boolean;
+  /** Image or icon to be displayed on the let side of the card */
+  image?: ReactNode;
+  /** Main card actions **/
+  actions?: ReactElement[];
+  /** Right-side actions */
+  secondaryActions?: ReactElement[];
+  /** Link to redirect to on card click. If provided, the Card inner content will be rendered inside `a` */
+  href?: string;
+  /** On click handler for the Card */
+  onClick?: () => void;
+}
+
+/**
+ * Generic card component
+ *
+ * @public
```

Comment:
can we change this to alpha so we can play around with it more before we mark this a beta or public